### PR TITLE
[Console] Use "\n" instead of PHP_EOL

### PR DESCRIPTION
**Pull request by ostrolucky**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        | 

"\n\r" is not needed in windows console either
screenshot from 2018-10-19 10-30-27

In fact, Windows will handle newlines correctly even in such a simple C++ code like following
```cpp
#include "pch.h"
#include <iostream>

int main()
{
    std::cout << "Hello\nWorld!"; 
}
```

Advantages of this change:
* Less bytes written to stream
* Less complex testcases thanks to consistent input/output
* Proves it's same in all environments which in turn weakens need for current abstraction in form of boolean flags for newline. This opens up possibility to deprecate them in future.

### CODE REVIEW

**Comment by iltar** [comment]
Is this also the case when you write to a file and then open the file in say notepad? I'm referring to the old notepad, not the upcoming one that supports linux file-endings.

File: src/Symfony/Component/Console/Tester/TesterTrait.php [hunk 1]
```diff
@@ -29,45 +29,29 @@
     /**
      * Gets the display returned by the last execution of the command or application.
      *
-     * @param bool $normalize Whether to normalize end of lines to \n or not
-     *
      * @return string The display
      */
-    public function getDisplay($normalize = false)
+    public function getDisplay()
```
Comment (stof) [inline src/Symfony/Component/Console/Tester/TesterTrait.php]: this feature should be deprecated, not removed, as some code might still be using `PHP_EOL` when writing messages, and so this logic might still be needed for a full migration.

**Review by stof — CHANGES_REQUESTED** [review]

**Comment by ostrolucky** [comment]
No, notepad.exe doesn't support "\n". And we shouldn't bend Symfony code just to accommodate it. Especially in **Console** component.

**Comment by nicolas-grekas** [comment]
I would like to do this change but @iltar argument make sense. Using a command and pipe it to a file is very common. Opening it using eg notepad is sometimes the only option you have and we managed to have this workflow work. I'm not sure we should break that even in 2018..

**Comment by javiereguiluz** [comment]
@ostrolucky a few days ago Microsoft made an historic announcement about Notepad and line endings (https://blogs.msdn.microsoft.com/commandline/2018/05/08/extended-eol-in-notepad/). So maybe this has already been fixed?

**Comment by iltar** [comment]
@javiereguiluz it's not fixed yet and it will take a while for this notepad to be used by the majority. My guess is the same adoption rate as IE/Edge vs FF/Chome. The problem is not limited to notepad however. I can imagine that windows applications that expect a CSV (generated via output direction), could also break if this is changed.

As long as windows doesn't change the their line endings to be `\n` as well, I don't think it's a good idea to remove support for them, regardless of what the console does. In fact, line-endings on the OS is something you should not worry about as end-user, so Symfony should do that for us.

**Comment by ostrolucky** [comment]
Do you know about some other application expecting `\n\r` other than notepad.exe?

**Comment by iltar** [comment]
@ostrolucky I don't know any, but I can tell you that there will be companies that wrote company private windows applications that will expect windows line endings instead of linux.

**Pull request closed without merging**